src/main/java/com/codecool/virtualpub/logic/Game.java

```diff
@@ -1,36 +1,86 @@
 package com.codecool.virtualpub.logic;
 
-import com.codecool.virtualpub.data.Bartender;
-import com.codecool.virtualpub.data.Customer;
-import com.codecool.virtualpub.data.Drink;
-import com.codecool.virtualpub.data.GameState;
+import com.codecool.virtualpub.data.*;
+import com.codecool.virtualpub.ui.Display;
+import com.codecool.virtualpub.ui.Input;
 
 import java.util.ArrayList;
 import java.util.List;
 
 public class Game {
-    public void Game() {
+
+    private Input input = new Input();
```

Comment:
Getting Input and Display from outside of the class ensures that we can easily change them if we need another kind of Input or Display. Right now they are hard-coded which means we have to change this class if some changes are needed with the Display or Input